src/client/chrome/commands.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+
+const {
+  toServerLocation,
+  fromServerLocation,
+  createLoadedObject,
+} = require("./create");
+
+import type { Location } from "../types";
+import type { ServerLocation, Agents } from "./types";
+
+type setBreakpointResponseType = {
+  breakpointId: string,
+  serverLocation?: ServerLocation,
+};
+
+let debuggerAgent;
+let runtimeAgent;
+let pageAgent;
+
+function setupCommands({ Debugger, Runtime, Page }: Agents) {
+  debuggerAgent = Debugger;
+  runtimeAgent = Runtime;
+  pageAgent = Page;
+}
+
+function resume() {
+  return debuggerAgent.resume();
+}
+
+function stepIn() {
+  return debuggerAgent.stepInto();
+}
+
+function stepOver() {
+  return debuggerAgent.stepOver();
+}
+
+function stepOut() {
+  return debuggerAgent.stepOut();
+}
+
+function pauseOnExceptions(
+  shouldPauseOnExceptions: boolean,
+  shouldIgnoreCaughtExceptions: boolean
+) {
+  if (!shouldPauseOnExceptions) {
+    return debuggerAgent.setPauseOnExceptions({ state: "none" });
+  }
+  const state = shouldIgnoreCaughtExceptions ? "uncaught" : "all";
```

Comment:
Maybe not in the scope of this review, but it would be nice if the `state` possible values (`none`, `uncaught`, `all`, ..) were in constants somewhere